### DISCUSSION TITLE
macOS Sequoia Beta installation - possible?

### DISCUSSION THREAD

**Discussion by MacUser341** [daliansky/XiaoMi-Pro#757] [General]
Hi
Can I use the same installation process for macOS Sequoia? The latest beta? 
Is there an option to install the native updates to the beta like in the original MacBook? 
I know its a beta and I understand it but I have my Xiomai notebook on the shelf with no use and I want the challenge.

**Comment by stevezz3** [comment]
@MacUser341 
For now, I don't recommend upgrading to Sequoia on XiaoMi Pro because
- Intel Wi-Fi is not ready, waiting for [itlwm](https://github.com/OpenIntelWireless/itlwm)'s update.
- Trackpad is not ready, waiting for [VoodooI2C](https://github.com/VoodooI2C/VoodooI2C)'s update.
- ...

As long as those major drivers provide support for macOS Sequoia, I will update the EFI release. Upon that, users should be good to perform a normal update to Sequoia like real Macs (Full backup is highly recommended as always).

**Comment by MacUser341** [comment]
Thanks. And if I have the Mac wifi card installed on my Xioami? And use a Bluetooth mouse? 
Cam I use the original EFI folder I have downloaded from here and build a SequoiaUSB stick ?

**Comment by stevezz3** [comment]
@MacUser341 Apple dropped support for many Broadcom wireless cards since Sonoma, including BCM943602CS recommended by us. Users have to use [OCLP](https://github.com/dortania/OpenCore-Legacy-Patcher) to modify their system partition and add network support back, which so far does not support Sequoia.

If you neglect Wi-Fi and trackpad functionality, you can update all the drivers and OpenCore manually based on the latest EFI release and then upgrade to Sequoia, or you can wait me updating the EFI release.

**Comment by MacUser341** [comment]
Thanks for your response, I will wait for you to update the EFI folder , hope it will be soon 

again thanks

**Comment by stevezz3** [comment]
@MacUser341 No problem, but it will take some time for Intel Wi-Fi to support Sequoia. I have to wait for all drivers support Sequoia.

**Comment by xjh169** [comment]
今天我更新了正式版 Sequoia 15.0 (24A335)，除了Wi-Fi和蓝牙 不能工作其他功能都正常了。使用有线网卡代替Wi-Fi，蓝牙几乎用不上。 😊

**Reply by MacUser341** [reply]
> 今天我更新了正式版 Sequoia 15.0 (24A335)，除了Wi-Fi和蓝牙 不能工作其他功能都正常了。使用有线网卡代替Wi-Fi，蓝牙几乎用不上。 😊

It’s even if I replaced the WiFi card to the one Apple uses ?

**Reply by stevezz3** [reply]
> > 今天我更新了正式版 Sequoia 15.0 (24A335)，除了Wi-Fi和蓝牙 不能工作其他功能都正常了。使用有线网卡代替Wi-Fi，蓝牙几乎用不上。 😊
> 
> It’s even if I replaced the WiFi card to the one Apple uses ?

Apple dropped support for any WiFi card you can purchase. They have to use OCLP to modify system volume to add back broadcom wifi drivers, which is highly risky.

**Comment by xjh169** [comment]
![11726557635_ pic_hd](https://github.com/user-attachments/assets/954b3f94-972f-4e2e-96ee-cdee50618a8f)
![截屏2024-09-17 15 22 04](https://github.com/user-attachments/assets/f5e5c6e6-3601-4342-878f-c9f8b58db1f0)
我不太懂。 @MacUser341
